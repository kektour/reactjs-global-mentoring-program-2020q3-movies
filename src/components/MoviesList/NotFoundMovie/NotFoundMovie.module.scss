@import 'src/styles/shared.scss';

.root {
  flex-grow: 1;
  display: flex;
}

.container {
  @extend %container;
  margin: auto;
}

.title {
  margin: 0;
  text-align: center;
  color: #fff;
  font-weight: normal;
  font-size: 50px;
  letter-spacing: 2px;
}
