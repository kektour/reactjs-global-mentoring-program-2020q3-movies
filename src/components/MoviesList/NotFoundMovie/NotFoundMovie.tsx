import React from 'react';
import styles from './NotFoundMovie.module.scss';

export const NotFoundMovie: React.FC<{}> = () => {
  return (
    <div className={styles.root}>
      <div className={styles.container}>
        <h1 className={styles.title}>No movie found</h1>
      </div>
    </div>
  );
};
