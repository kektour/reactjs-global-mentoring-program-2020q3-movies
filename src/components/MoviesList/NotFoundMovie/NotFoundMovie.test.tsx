import React from 'react';
import { render } from '@testing-library/react';
import { NotFoundMovie } from './NotFoundMovie';

describe('NotFoundMovie', () => {
  it('Should render component correctly', () => {
    const component = render(<NotFoundMovie />);
    expect(component.asFragment()).toMatchSnapshot();
  });
});
