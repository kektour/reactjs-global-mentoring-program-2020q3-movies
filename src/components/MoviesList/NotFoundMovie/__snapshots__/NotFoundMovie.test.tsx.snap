// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundMovie Should render component correctly 1`] = `
<DocumentFragment>
  <div
    class="root"
  >
    <div
      class="container"
    >
      <h1
        class="title"
      >
        No movie found
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
